azurerm/resource_arm_managed_disk.go

```diff
@@ -102,6 +102,18 @@ func resourceArmManagedDisk() *schema.Resource {
 			"encryption_settings": encryptionSettingsSchema(),
 
 			"tags": tagsSchema(),
+
+			"disk_iops_read_write": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+			},
+
+			"disk_mbps_read_write": {
+				Type:     schema.TypeInt,
+				Optional: true,
```

Comment:
Same here, I personally would remove the `Optional` attribute from this parameter since it is a `read-only` property of the resource... what do you think?

Reply:
I think this property is not read-only. Because it can be specified to update the bandwidth while using ultra ssd disk. You can refer the document (https://docs.microsoft.com/en-us/rest/api/compute/disks/createorupdate#creationdata) and search by keyword "properties.diskMBpsReadWrite" which describes the usage of this property.